src/main/scala/com/databricks/labs/overwatch/pipeline/Gold.scala

```diff
@@ -407,6 +474,7 @@ class Gold(_workspace: Workspace, _database: Database, _config: Config)
     restoreSparkConf()
     executeModules()
     buildFacts()
+    publishVerbose()
```

Comment:
There's no try/catch here at all -- so if something were to fail in this function it would break the pipeline.